addons/map/README.md

```diff
@@ -8,6 +8,10 @@ Various tweaks to the in-game map, including:
 - Map illumination (optional)
 - Blue Force Tracker (optional)
 
+#### Variables
```

Comment:
We don't include documentation like this in Readme file. Please commit it to the documentation branch instead (`gh-pages`).  The ace3mod.com web site is auto generated from the content of that branch,